src/cmds/eval/evalWorker.ts

```diff
@@ -0,0 +1,57 @@
+/* global lockdown, Compartment, BigInt */
+
+import { parentPort } from 'worker_threads';
+import { readFileSync } from 'fs';
+import cryptography from 'crypto';
+
+require('ses/lockdown');
```

Comment:
You're going to have to add back the `eslint-disable` comment once linting is enabled.